README.md

```diff
@@ -256,22 +267,91 @@ kpt alpha repo register \
+Workload clusters must run Config Sync to get their workloads from their deployment repositories.
 
 The package
-[nephio-configsync](https://github.com/nephio-project/nephio-packages.git/nephio-configsync)
+[nephio-configsync](https://github.com/nephio-project/nephio-packages/tree/main/nephio-configsync)
 is intended for installing Config Sync in those clusters. To use this package,
 you will need to update the RootSync resource with the repository and
 authentication needed for your environment.
 
 Example:
 ```
-kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages.git/nephio-configsync test-deploy-edge-01
-kpt fn render test-deploy-edge-01
-kpt live init test-deploy-edge-01
-kpt live --context=gke_jbelamaric-dev_us-central1_nephio-edge-01 apply test-deploy-edge-01 --reconcile-timeout=15m --output=table
+kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages.git/nephio-configsync nephio-test-deploy-01
+# Replace the repository name in rootsync.yaml with the $GITHUB_USERNAME
+kpt fn eval nephio-test-deploy-01 \
+  --save \
+  --type mutator \
+  --image gcr.io/kpt-fn/search-replace:v0.2.0 \
+  -- by-path=spec.git.repo by-value-regex='https://github.com/[a-zA-Z0-9-]+/(.*)' \
+  put-value="https://github.com/${GITHUB_USERNAME}/\${1}"
+kpt fn render nephio-test-deploy-01
+kpt live init nephio-test-deploy-01
+kpt live apply nephio-test-deploy-01 --reconcile-timeout=15m --output=table
+
+# Verify that the config sync pods are running
+kubectl get pods --all-namespaces | grep config
+config-management-monitoring   otel-collector-5b8d5d8d8f-8qs2k                                 1/1     Running   0          1m
+config-management-system       config-management-operator-6d7867f9f5-xb9cn                     1/1     Running   0          3m
+config-management-system       reconciler-manager-c6c8cf7f6-9x7vr                              2/2     Running   0          1m
+config-management-system       root-reconciler-nephio-workload-cluster-sync-679dd89788-mkl8x   4/4     Running   0          1m
+```
+
+## Deploying a Package Workload
+We have two options to provision a sample workload.
+  * Use the Nephio Web UI in case you followed the steps in [Installing the Nephio Web UI](#installing-the-web-ui) above, or
+  * Use `kpt` and `git` to provision the workload from the command line.
+
+[Installing the Nephio Web UI](#installing-the-web-ui) (as described above) was optional.
+Therefore, we will use `kpt` and `git` to provision the workload.
+
+Example:
+```
+git clone https://github.com/${GITHUB_USERNAME}/nephio-test-deploy-01.git caching-dns
+cd caching-dns
+
+# Get the package for deployment
+kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages/tree/main/coredns-caching-scaled scaled
+kpt fn render scaled
+
+# Check out the rendered package
+kpt pkg tree scaled
+Package "scaled"
+├── [Kptfile]  Kptfile scaled
+├── [clusterscaleprofile.yaml]  ClusterScaleProfile scale-profile
```

Comment:
This is great as an example of how to do it directly through Git.

There is also a CLI journey that we can show that uses Porch, and avoids the need to touch Git at all. Roughly the steps are:

```
# Find the right packagerevision name
$ kpt alpha rpkg get
NAME                                                           PACKAGE                  REVISION   LATEST   LIFECYCLE   REPOSITORY
nephio-packages-e01d890d4c85fc62299d956829ffe948d712bd76       coredns-caching          main       false    Published   nephio-packages
nephio-packages-edfea244e9255e476de3dcc00b56003104f1d4cd       coredns-caching          v1         true     Published   nephio-packages
nephio-packages-25e81190f7cc80d4ccc830b7542b979cd4f6168e       coredns-caching-scaled   main       false    Published   nephio-packages
kpt alpha rpkg clone nephio-packages-8f6ade1048c5a7da0d8364d5c82026339f4950ad dnscache --repository nephio-dev-deploy -n default

## Clone it from the catalog repo to the deployment repo
$ kpt alpha rpkg clone nephio-packages-8f6ade1048c5a7da0d8364d5c82026339f4950ad dnscache --repository nephio-test-deploy-01 -n default

## Propose the package
$ kpt alpha rpkg propose test-deploy-edge-01-93bf2ce0378cf033c74ae5296eb93585f54bea09 -n default

## Approve the package (publish it)
$ kpt alpha rpkg approve test-deploy-edge-01-93bf2ce0378cf033c74ae5296eb93585f54bea09 -n default
```
That should be equivalent to the Git commands, but avoids Git, and tracks the metadata in the same way that the GUI would.

What I did in the demo went a step further, and had a controller propose each of these packages, and inject a unique ClusterScaleProfile associated with the cluster it is being deployed to. But that needs more work on the controller to package it and clean it up a bit more, then run it alongside porch (embed it in `nephio-system` package).

Reply:
Awesome! I really like the pure kpt approach you are proposing here. I have modified the README accordingly.